victor/Cargo.toml

```diff
@@ -21,6 +21,7 @@ lock_api = "0.1"
 matches = "0.1"
 num-traits = "0.2"
 parking_lot = "0.6"
+rayon = "1"
```

Comment:
Nit: is this the same as `rayon = "1.0"`?

Reply:
It is.